composables/transaction/transactionBuy.ts

```diff
@@ -9,8 +9,27 @@ import { getApiCall } from '@/utils/gallery/abstractCalls'
 import { payRoyaltyTx, somePercentFromTX } from '@/utils/support'
 import type { ActionBuy } from './types'
 import { verifyRoyalty } from './utils'
+import { existentialDeposit } from '@kodadot1/static'
+import { decodeAddress, encodeAddress } from '@polkadot/util-crypto'
+const formatAddress = (address: string): string => {
+  const { chainProperties } = useChain()
 
-function payRoyaltyAssetHub(
+  const publicKey = decodeAddress(address)
+  return encodeAddress(publicKey, chainProperties.value.ss58Format)
+}
+
+const getFallbackAddress = () => {
+  // subtrate format
+  // TODO:
+  //  1.change to real royalty recipient
+  //  2. move to static?
+
+  const FALBACK_ROYALTY_RECIPIENT =
+    '5EUDt8Gu6e1tQozggApnHbzFTqwGwZs7696Hupk595XTa2Ht'
```

Comment:
Whose address is this?

Please use different one 
Cc @yangwao

Reply:
It's mine.
Obviously use different one
Lmk which

Reply:
It's mine.
Obviously use different one
Lmk which

Reply:
Sorry, could not find any of these

I am familiar with Wayland fund and kodadot minter

https://polkadot.subscan.io/account/15CoYMEnJhhWHvdEPXDuTBnZKXwrJzMQdcMwcHGsVx5kXYvW


Mind providing an address?

Reply:
All if defined here `utils/support.ts`